internal/riverinternaltest/riverdrivertest/riverdrivertest.go

```diff
@@ -905,12 +905,12 @@ func Exercise[TTx any](ctx context.Context, t *testing.T,
 				require.Nil(t, job.AttemptedAt)
 				require.Empty(t, job.AttemptedBy)
 				require.WithinDuration(t, now, job.CreatedAt, 2*time.Second)
-				require.Equal(t, []byte(`{"encoded": "args"}`), job.EncodedArgs)
+				require.JSONEq(t, `{"encoded": "args"}`, string(job.EncodedArgs))
```

Comment:
I can already tell it's going to drive me crazy that they didn't called this `JSONEqual` to match all the other existing `Equal` helpers.